Assets/HoloToolkit/SharingWithUNET/Scripts/UNetSharedHologram.cs

```diff
@@ -53,7 +53,7 @@ private void Start()
 #if UNITY_2017_2_OR_NEWER
             isOpaque = UnityEngine.XR.WSA.HolographicSettings.IsDisplayOpaque;
 #else
-            isOpaque = Application.isEditor;
+            isOpaque = !UnityEngine.VR.VRDevice.isPresent;
```

Comment:
Another change from `Application.isEditor` -> `VRDevice.isPresent`

Reply:
This is also outdated. The current version is always written as suggested.